src/services/ServerSettingsService.ts

```diff
@@ -39,9 +39,13 @@ export default class ServerSettingsService {
       const adminUser = await new UserService()
         .createAdminUser(admin);
 
-      new AuthService().createIdentityLocal(adminUser!, ldapEnabled());
+      const authService: AuthService = new AuthService();
+
+      const $identity = await authService.createIdentityLocal(adminUser!, ldapEnabled());
 
       await this.setSetting({ name: 'SETUP_DONE', value: 'true' });
+      return authService.getSetPasswordToken(adminUser!, $identity);
```

Comment:
If I read this correctly, the `initialSetup` method now returns a token to reset your password? I think it is better to skip this extra back-and-forth step all together. Instead, you can add a "password" field to the `SetupParams`. Then, you can immediately set the password for the newly created user, so extra steps from the frontend and the enduser a not necessary.

If you want to make it even more fancy, you can also directly authenticate the user. But if you feel that is out of scope for this PR, I respect that.